domain/consensus/model/externalapi/consensus.go

```diff
@@ -13,9 +13,11 @@ type Consensus interface {
 
 	GetHashesBetween(lowHash, highHash *DomainHash, maxBlueScoreDifference uint64) ([]*DomainHash, error)
 	GetMissingBlockBodyHashes(highHash *DomainHash) ([]*DomainHash, error)
-	GetPruningPointUTXOSet(expectedPruningPointHash *DomainHash) ([]byte, error)
+	GetPruningPointUTXOs(expectedPruningPointHash *DomainHash, offset int, limit int) ([]*OutpointAndUTXOEntryPair, error)
 	PruningPoint() (*DomainHash, error)
-	ValidateAndInsertPruningPoint(newPruningPoint *DomainBlock, serializedUTXOSet []byte) error
+	ClearImportedPruningPointData() error
+	InsertImportedPruningPointUTXOs(outpointAndUTXOEntryPairs []*OutpointAndUTXOEntryPair) error
```

Comment:
Maybe change this to `AccumulatePruningPointUTXO` or append instead of accumulate